### DISCUSSION TITLE
How can I control the space between persona elements inside a form card

### DISCUSSION THREAD

**Discussion by VijithaEkanayake-zz** [h2oai/wave#1045] [Q&A]
In my application persona is used in a FormCard inside a of ui.inline as below. However, I cannot find a way to control the space between these persona elements? How can I control the width of the persona element?

```python
 ui.inline(
      items=[
          ui.persona(name="persona", title='', size='xs', image=q.app.app_images['share']),
          ui.persona(title='', size='xs', image=image),
          ui.persona(title='', size='xs', image=image),
          ui.persona(title='', size='xs', image=image),
          ui.persona(title='', size='xs', image=image),
      ]
  )
```

Below is the screenshot of the card which contains more space between personas
<img width="517" alt="Screenshot 2021-10-05 at 18 23 11" src="https://user-images.githubusercontent.com/5285626/136040452-fbeb7123-794a-4c67-8ffd-cdf8cc1135a5.png">

Need to implement the card with minimum space between persona elements as below.

<img width="345" alt="Screenshot 2021-10-05 at 19 55 16" src="https://user-images.githubusercontent.com/5285626/136042750-576fdae3-56f0-46ee-a477-37cf45f65cf1.png">

**Comment by mturoci** [comment]
Can add `width` prop to `ui.persona`, but it will not solve the whole problem. A new facepile component (https://developer.microsoft.com/en-us/fluentui?fabricVer=8#/controls/web/facepile) needs to be implemented to allow for auto overflow handling and add functionality.

Tracked in: https://github.com/h2oai/wave/issues/1047
